src/hooks/useTenderProposals.ts

```diff
@@ -17,5 +17,7 @@ export function useTenderProposals(
     tenderProposals: proposals,
     isLoadingTenderProposals: isLoadingProposals,
     hasTenderProcessStarted: !!proposals?.data && hasTenderProcessStarted(proposals?.data),
+    hasTenderProcessFinished: !!proposals?.data && hasTenderProcessFinished(proposals?.data),
```

Comment:
`hasTenderProcessFinished` & `hasTenderProcessStarted` could allow for null|undefined params and return false for those cases, so we can skip checking when calling them